### DISCUSSION TITLE
Cannot detect that project is already a Git repository.

### DISCUSSION THREAD

**Discussion by neemazephania** [IBAHCM/RPiR#239] [Archive] [answered]
Hi @soniamitchell, while setting up git repository I run into this error. 

<img width="1000" alt="Screenshot 2021-02-05 at 14 16 35" src="https://user-images.githubusercontent.com/62024296/107045644-a767a700-67bd-11eb-9efa-b5d925f18435.png">

I don't even know how to deal with it.

**Comment by soniamitchell** [comment]
Hey @neemazephania, why are you using `gitr2::init()`? 

Could you try committing your files, then running `usethis::use_github(organisation = "IBAHCM", private = TRUE)`?

**Reply by neemazephania** [reply]
I did committed them in the first place and then after this command `usethis::use_github(organisation = "IBAHCM", private = TRUE)` I was getting an error "cannot detect that project is already a git repo"

so I decide to try couple of online solution and one was to use `gitr2::init()` to check if I really committed files, and it does shows the initial commit I made. 

then after I didn't know what to do anymore.

I also tried to delete project and recreate again but the error still exists.

**Reply by soniamitchell** [reply]
what happens if you run `usethis::use_git()` ? (if it asks you to commit stuff, say yes)

**Comment by neemazephania** [comment]
I'm getting this 

<img width="1005" alt="Screenshot 2021-02-06 at 11 52 17" src="https://user-images.githubusercontent.com/62024296/107117365-d5a6be80-6871-11eb-8a88-b61d2f9024e8.png">

**Reply by soniamitchell** [reply]
If you restart R, then make a package with a slightly different name, e.g. `neemazephaniaSeries3`, then make sure you commit first before running `usethis::use_github(organisation = "IBAHCM", private = TRUE)` do you still get an error?

**Reply by neemazephania** [reply]
same error 

<img width="1013" alt="Screenshot 2021-02-06 at 12 08 12" src="https://user-images.githubusercontent.com/62024296/107117706-1a335980-6874-11eb-9275-acd6d49cc344.png">

**Comment by soniamitchell** [comment, marked as answer]
Could you try upgrading Xcode? https://ma.ttias.be/mac-os-xcrun-error-invalid-active-developer-path-missing-xcrun/

**Reply by neemazephania** [reply]
now I'm getting this but at least : )

<img width="1000" alt="Screenshot 2021-02-06 at 12 21 21" src="https://user-images.githubusercontent.com/62024296/107117991-f244f580-6875-11eb-8528-41a26cc3bbad.png">

**Reply by neemazephania** [reply]
I tried to use another computer and everything went as planned, my computer could be a problem here.

**Reply by soniamitchell** [reply]
Cool! It looks like your PAT might have a problem. Could you do this for me? https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat

**Reply by neemazephania** [reply]
Thanks a lot @soniamitchell , everything is working perfectly fine now.😊

**Reply by soniamitchell** [reply]
Woo! Happy to help 😄
